src/ui_basic/editbox.cc

```diff
@@ -212,6 +213,55 @@ bool EditBox::handle_mousepress(const uint8_t btn, int32_t, int32_t) {
 	return false;
 }
 
+void EditBox::set_caret_to_cursor_pos(int32_t x) {
+	if (x <= 2 * kMarginX) {
```

Comment:
when / how can this happen?

Reply:
this happens when you click on the leftmost spot (left border) of the widget. x=0 is not the start of the first char (there is a margin before the text begins).